image_tools_py/README.md

```diff
@@ -0,0 +1,106 @@
+## Image Tools Demo in Python
```

Comment:
@clalancette watch out for the one sentence per line policy (see [markdown style guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#markdown-restructured-text-docblocks), third bullet).

Reply:
Fixed throughout the document in 2335206